rosdep/python.yaml

```
@@ -7415,6 +7415,28 @@ python3-pykdl:
     '*': [python3-pykdl]
     bionic: null
     xenial: null
+python3-pylibdmtx:
+  debian:
+    '*':
+      packages: [python3-pylibdmtx]
```

For rules that utilize the system's default package manager it isn't necessary to use the nested 'packages' map. Same change applies to the Ubuntu `*` rule.
```suggestion
    '*': [python3-pylibdmtx]
```

---

@alexMilmore Friendly ping. After addressing this comment, I think we can merge :)